jquery.i18n/languages/en.json

```diff
@@ -0,0 +1,57 @@
+{
+	"@metadata": {
+		"authors": ["JustinCast"],
+    "last-updated": "2020-03-20",
+    "locale": "en"
+	},
+	"title": "Jungle TestNet",
```

Comment:
@JustinCast  let's replace gthis with : `"title" :  "Jungle Testnet - The best place to test your EOS DApps",`